python/ray/serve/api.py

```diff
@@ -571,3 +577,102 @@ def delete(name: str, _blocking: bool = True):
     """
     client = get_global_client()
     client.delete_apps([name], blocking=_blocking)
+
+
+@PublicAPI(stability="alpha")
+def multiplexed(func=None, num_models_per_replica: int = 0):
+    """Coverts a function or method to a multiplexed function.
```

Comment:
Can you specify the considerations about `async def` vs `def` in this docstring please?